.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+## Changes description
+<!--- Describe your changes in details. -->
+
+## Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an opened issue, please link to the issue here. -->
+
+## Checklist
+<!--- Feel free to add other steps if needed. -->
+
+- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
+- [ ] Check the commit's or even all commits' message styles matches our requested structure.
```

Comment:
should we also add the link to 'conventinalcommits.org' ? Or is there any other description of the expected structure that we could link to ?

Reply:
you're right, fixed!